addons/payment_stripe/static/src/js/express_checkout_form.js

```diff
@@ -180,6 +175,7 @@ paymentExpressCheckoutForm.include({
                         },
                     },
                 );
+                const availableCarriers = availableCarriersData.delivery_methods;
```

Comment:
nitpicking: you can also unpack values directly if you'd like
```suggestion
                const { delivery_methods, delivery_discount_minor_amount } = availableCarriersData.delivery_methods;
```